LEAF_Nexus/admin/templates/setup_medical_center.tpl

```diff
@@ -563,6 +563,9 @@ $(function() {
 	    case 'services':
             setupServices();
             break;
+        case 'primaryAdmin':
```

Comment:
need to get rid of need to be case sensitive.

Reply:
Added case sensitivity to the switch variable